gitlab-ci: Allow to select a runner for a schedule pipeline

lucas-p-bernardes opened the pull request:
When RUNNER_TAG variable is defined in a schedule pipeline, the pipeline will execute the jobs on the runner that contains the tag defined. If not defined, will execute jobs on untagged runner.

Related-to: TCB-253

jsrc27 reviewed (approved):
LGTM

rborn-tx reviewed (commented):
@lucasbernardestoradex Have you tried setting a tag for all jobs? (as mentioned [here](https://stackoverflow.com/a/77604988))

lucas-p-bernardes committed:
gitlab-ci: Allow to select a runner for a schedule pipeline

When RUNNER_TAG variable is defined in a schedule pipeline, the pipeline
will execute the jobs on the runner that contains the tag defined. If
not defined, will execute jobs on untagged runner.

Related-to: TCB-253

Signed-off-by: Lucas Bernardes <lucas.bernardes@toradex.com>

lucas-p-bernardes commented:
> @lucasbernardestoradex Have you tried setting a tag for all jobs? (as mentioned [here](https://stackoverflow.com/a/77604988))

I tried this and works very well. Thanks for the tip!
I pushed the changes with the suggestion.

lucas-akira reviewed (approved):
LGTM

lucas-p-bernardes commented:
@rborn-tx @lucas-akira Can we have this merged?

lucas-akira commented:
On my side, yes, looks good to me.
